Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -6215,6 +6215,36 @@ def get_anti_spyware_best_practice_command():
     })
 
 
+def apply_dns_signature_policy_command(args) -> CommandResults:
+    anti_spy_ware_name = args.get('Anti_spyware_profile_name')
+    edl = args.get('DNS_signature_source')
+    action = args.get('Action')
+    packet_capture = args.get('Packet_capture', 'disable')
+    params = {
+        'action': 'set',
+        'type': 'config',
+        'xpath': f"/config/devices/entry[@name='localhost.localdomain']/device-group/entry[@name='{DEVICE_GROUP}']"
+                 f"/profiles/spyware/entry[@name='{anti_spy_ware_name}']",
```

Comment:
```suggestion
        'xpath': f"{XPATH_OBJECTS}/profiles/spyware/entry[@name='{anti_spy_ware_name}']",
```
XPATH_OBJECTs should refer to the correct device group already. 

![image](https://user-images.githubusercontent.com/53861351/163969673-7e914353-9941-4b11-b768-3dd37da098c5.png)

see for example: 
```
@logger
def panorama_delete_custom_url_category(custom_url_category_name: str):
    params = {
        'action': 'delete',
        'type': 'config',
        'xpath': XPATH_OBJECTS + "profiles/custom-url-category/entry[@name='" + custom_url_category_name + "']",
        'element': "<entry name='" + custom_url_category_name + "'></entry>",
        'key': API_KEY
    }
    result = http_request(
        URL,
        'POST',
        body=params,
    )

    return result
```

Reply:
if you will look closely the prefix you want to change is not the same as in the XPATH_OBJECTS  'localhost.localdomain' is missing. this prefix is similar to others that i saw dealing with the anti-spyware commands.

Reply:
did you try it with my suggestion? check if it works, if it does not keep it the way you did.

Reply:
It dosent work your way i tried it. keeping the original code here